README.md

```diff
@@ -55,11 +55,12 @@ mapped onto the src directory.
 
 The primary thing provided by this library are the Serializer and Deserializer namespaces.
 A set of Exceptions each process typically can encounter are also provided, and are located
-in respective Exceptions namespaces. They all derive from (Des/S)erializationException.
+in respective Exceptions namespaces. They all derive from
+`SerializationException`/`DeserializationException`.
 
-Both interfaces define two methods: a (de)serialize method that does the actual work, and
-a can(Des/S)erialize method that allows finding out if a given (de)serializer can process
-a given input.
+The main interfaces define the `serialize`/`deserialize` methods that do the actual work.
+In addition there is an interface with a `isSerializerFor`/`isDeserializerFor` method that allows
```

Comment:
I would phrase this as “there are interfaces” rather than “there is an interface”, since the previous sentence also talks about “the main interfaces” in the plural.

Reply:
Word. I did the change in the github GUI and after I read it in the PR thought the same thing - but as (to my knowledge) there is no way to amend I decided to flip the coin.